packages/shared/components/NodeConfigurationForm.svelte

```diff
@@ -175,5 +175,7 @@
         label={localize('popups.node.optionalJwt')}
         disabled={isBusy}
     />
-    <Error error={formError} />
+    {#if formError}
```

Comment:
adding the check in the component would mean being able to remove this check here